### PR TITLE
Photo updates

### PR DESCRIPTION
**Pull request by lawrenbd**
Including the updated images running in the local server; haven't updated gifs yet. Please confirm that I handled the commits correctly, otherwise I felt fine updating the code.

Homepage laptop image update
image

Features - Document
image

Features - Training
image

Features - Issues
image

Features - Audits
image

### CODE REVIEW

**Comment by lawrenbd** [comment]
@leonardorb, for the Jekyll local server, for your reference, I figured out that I can just run a "jekyll serve" command and it launches the local server

File: client-support-associate.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-permalink: /client-support-associate/
+permalink: /csupport/
```
Comment (leonardorb) [inline client-support-associate.md]: @lawrenbd `client-support-associate` will definitely help us with SEO here. Could you keep it?

File: test.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ permalink: /test/
       </div>
     </div>
     <div class="homepage-hero-right">
-      <img src="/img/home/hero-img.png" alt="zen-dash">
+      <img src="/img/home/laptop-new-dash" alt="zen-dash">
```
Comment (leonardorb) [inline test.md]: Please add the image extension here.

**Comment by lawrenbd** [comment]
@leonardorb, made the changes. Need to work on my coding attention to detail!

**Pull request merged**